src/services/api.js

```diff
@@ -1,11 +1,73 @@
-import axios from "axios";
+import axios from 'axios';
 
-const API_BASE_URL = "http://localhost:8000/api";
+const API_BASE_URL = 'http://localhost:8000/api';
```

Comment:
I thought we had made this an envi variable. if not, we should add it again or use it.